openupgrade_scripts/scripts/project/16.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2023 Trần Trường Sơn
+# Copyright 2023 Rémy Taymans
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import openupgradelib as openupgrade
```

Comment:
It should be `from openupgradelib import openupgrade`

Reply:
Indeed. Fixed. :)